master/internal/rm/kubernetesrm/pods.go

```diff
@@ -679,17 +694,29 @@ func (p *pods) deleteDoomedKubernetesResources(ctx *actor.Context) error {
 	return nil
 }
 
-func (p *pods) startPodInformer(ctx *actor.Context) {
+func (p *pods) startPodInformer(ctx *actor.Context) error {
 	for namespace := range p.namespaceToPoolName {
-		i, _ := ctx.ActorOf("pod-informer-"+namespace,
-			newInformer(p.podInterfaces[namespace], ctx.Self()),
-		)
+		i, err := newInformer(context.TODO(), namespace, p.podInterfaces[namespace])
+		if err != nil {
+			return errors.Errorf("pod informer for %s failed", namespace)
+		}
+		go i.startInformer(func(pod *k8sV1.Pod) { p.receivePodStatusUpdate(ctx, pod) })
```

Comment:
doesn't really matter here, since these things are always supposed to be up, but we aren't managing the lifetime of this goroutine